scripts/pi-hole/js/db_graph.js

```diff
@@ -196,6 +196,8 @@ function updateQueriesOverTime() {
         }
       }
 
+      timeLineChart.options.scales.xAxes[0].ticks.min = from * 1000;
```

Comment:
Why do we need to multiply by 1000 here?

Reply:
Because timestamps are in seconds, but javascript uses microseconds.

Reply:
Thanks.